### PR TITLE
Make variables for more parameters used in more locations

### PR DESCRIPTION
**Pull request by dillwishlist**
Change the save the date pages primarily to use the site settings variables.

### CODE REVIEW

**Comment by dillwishlist** [comment]
Also changes footer to give credit and link to original (this) repo.

File: bigday/settings.py [hunk 1]
```diff
@@ -126,19 +126,25 @@
...
+# base address for all emails
+DEFAULT_WEDDING_EMAIL = 'happilyeverafter@example.com'
 # the address your emails (save the dates/invites/etc.) will come from
-DEFAULT_WEDDING_FROM_EMAIL = 'You and Your Partner <happilyeverafter@example.com>'
+DEFAULT_WEDDING_FROM_EMAIL = BRIDE_AND_GROOM + ' ' + DEFAULT_WEDDING_EMAIL # change to 'address@domain.tld'
 # the default reply-to of your emails
-DEFAULT_WEDDING_REPLY_EMAIL = 'happilyeverafter@example.com'
+DEFAULT_WEDDING_REPLY_EMAIL = DEFAULT_WEDDING_EMAIL # change to 'address@domain.tld'
+# the location of your wedding
+WEDDING_LOCATION = 'North Pole, USA'
```
Comment (czue) [inline bigday/settings.py:138]: :snowman:

File: bigday/settings.py [hunk 2]
```diff
@@ -126,19 +126,25 @@
     os.path.join('bigday', 'static'),
 )
 
+# This is used in a few places where the names of the couple are used
+BRIDE_AND_GROOM = 'Bride and Groom'
+# base address for all emails
+DEFAULT_WEDDING_EMAIL = 'happilyeverafter@example.com'
 # the address your emails (save the dates/invites/etc.) will come from
-DEFAULT_WEDDING_FROM_EMAIL = 'You and Your Partner <happilyeverafter@example.com>'
+DEFAULT_WEDDING_FROM_EMAIL = BRIDE_AND_GROOM + ' ' + DEFAULT_WEDDING_EMAIL # change to 'address@domain.tld'
```
Comment (czue) [inline bigday/settings.py:134]: Is the `<>` enclosure not necessary? (admittedly I forget how this is used)

File: bigday/templates/base.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
     <meta name="description" content="">
     <meta name="author" content="">
     <!-- Modified from https://github.com/IronSummitMedia/startbootstrap-creative -->
-    <title>Cory and Rowena's big day</title>
+    <title>{{ support_email }}'s big day</title>
```
Comment (czue) [inline bigday/templates/base.html:11]: Should this not be `{{ couple_name }}`?

**Review by czue — APPROVED** [review]
This looks great, thanks for the contribution!

Had a few questions, but I'll go ahead and merge now since this is already an improvement. I should really make time to go back and improve this at some point...

File: bigday/templates/base.html (hunk 3 above)
Comment (dillwishlist) [reply bigday/templates/base.html:11]: Yes.  I fixed it in the other PR this morning.

File: bigday/settings.py (hunk 2 above)
Comment (dillwishlist) [reply bigday/settings.py:134]: Good point, I'll correct shortly.

**Pull request merged**